content/fa/docs/setup/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+no_issue: true
+title: setup
+main_menu: true
```

Comment:
nit: Does this front matter match the front matter in `content/en/docs/setup/_index.md`?